scripts/vaccination_alerts/utils.ts

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+import {
+  RegionVaccinePhaseInfo,
+  stateVaccinationPhases,
+} from '../../src/cms-content/vaccines/phases';
+
+export interface RegionVaccinePhaseInfoMap {
+  [fipsCode: string]: RegionVaccinePhaseInfo;
+}
+
+export interface RegionVaccineVersion {
+  emailAlertVersion: number;
+}
+
+export interface RegionVaccineVersionMap {
+  [fipsCode: string]: RegionVaccineVersion;
+}
+
+export function getCmsVaccinationInfo(): RegionVaccinePhaseInfoMap {
+  return _.chain(stateVaccinationPhases)
+    .map(stateInfo => [stateInfo.fips, stateInfo])
+    .fromPairs()
+    .value();
+}
+
+// TODO: Fetch this from Firebase, return an empty object if Firebase
+// doesn't have any data.
+export function getFirebaseVaccinationInfo(): RegionVaccineVersionMap {
+  return {
+    // '29': {
+    //   emailAlertVersion: 0,
+    // },
+  };
+}
+
+export function getUpdatedVaccinationInfo(
+  currentInfoByFips: RegionVaccinePhaseInfoMap,
+  lastVersionByFips: RegionVaccineVersionMap,
+): RegionVaccinePhaseInfoMap {
+  const updatedInfoByFips: RegionVaccinePhaseInfoMap = {};
+  for (const fipsCode in currentInfoByFips) {
+    const lastVersionInfo = lastVersionByFips[fipsCode];
+    const currentPhaseInfo = currentInfoByFips[fipsCode];
+
+    if (
+      !lastVersionInfo ||
+      lastVersionInfo.emailAlertVersion !== currentPhaseInfo.emailAlertVersion
+    ) {
+      updatedInfoByFips[fipsCode] = currentPhaseInfo;
+    }
+  }
+  return updatedInfoByFips;
+}
```

Comment:
I wanted to add unit tests for this function, but CRA only runs tests under `src` 😢

Reply:
Oh, hrm.  Maybe we should consider moving the scripts dir or something.

Reply:
scripts does feel a bit like the wild wild west - I'm not sure if we run type checks on it automatically and sometimes imports are not found cleanly.  Maybe moving scripts into src would fix taht?

Reply:
FWIW- Right now we have a separate tsconfig.json that specifies to generate CommonJS module code, in order for it to work with node.js... https://github.com/covid-projections/covid-projections/blob/3cfc1d8cc2e39c88224da919557f845df04581d1/scripts/tsconfig.json#L4

This will probably be an obstacle to moving the code into `src/` but maybe there's a solution.

I think the existence of the extra tsconfig.json file is also why you have to specify a relative path to imports instead of being able to base it off of `src/` (which I'm guessing is what you meant about clean imports).